index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <h1 class="name">John Doe</h1>
+        <nav class="hamburger"> 
+            <div></div>
+            <div></div>
+            <div></div>
+        </nav>
+    </header>
+<section class="headline">
+    <h2 class="title">Hey There.
```

Comment:
Please replace `<h2>` tag with `<h1>` as this is going to be your main heading that's it should be written in `<h1>` tag and there should be only one `<h1>` tag in a HTML document because of semantic reason.